framework/auth/core.py

```diff
@@ -210,7 +215,7 @@ class User(GuidStoredObject, AddonModelMixin):
         'github': u'http://github.com/{}',
         'scholar': u'http://scholar.google.com/citation?user={}',
         'twitter': u'http://twitter.com/{}',
-        'personal': u'{}',
+        'profileWebsites': u'[{}]',
```

Comment:
Should be fine. Any code that accesses/displays it will have to be able to deal with a list, but elasticsearch shouldn't mind. Don't think a migration will be required, but I'll have to look into it.